setup.py

```diff
@@ -63,7 +63,7 @@
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=[''],
+    install_requires=['pyzmq'],
```

Comment:
I have removed this file for the first PR, perhaps you can add the requirement  to a pip-requirements.txt?